connectors/sources/jira.py

```diff
@@ -822,7 +836,7 @@ async def _get_projects(self):
                     ):
                         project = await response.json()
                         await self._put_projects(project=project, timestamp=timestamp)
-            await self.queue.put("FINISHED")  # pyright: ignore
+            await self.queue.put(FINISHED)  # pyright: ignore
```

Comment:
Do we actually need pyright: ignore here after this change? What are we even ignoring here? :D